res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
These kinds of things won't work with the current OpenSUSE Leap for example. It is only running RPM Version 4.11.2 by default. It needs a specific package built for it.